content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -195,17 +195,6 @@ pool:
         latency: 5ms
 ```
 
```

Comment:
Since we do not have a versioned documentation, we need to mention somewhere in our documentation that they can check the old files in one of the older releases/tags /branches of the docs repo. 
when we have versioned documentation we can generate the docs from those respective tags or branches.

Reply:
@rajat-soda How do you propose we handle this?

Reply:
The only versioning we have now , is Git. Instead of releasing the Master branch of documentation repo, we can release the branch (Eg. Greenland, Hawaii etc) and use that for the documentation branch.